tests/ui/methods/disambiguate-multiple-blanket-impl.stderr

```diff
@@ -52,9 +52,9 @@ LL |     const CONST: usize = 2;
    |     ^^^^^^^^^^^^^^^^^^
 help: use fully-qualified syntax to disambiguate
    |
-LL |     <T as A>::CONST;
+LL |     <S as A>::CONST;
```

Comment:
This is more correct, since the code is now suggesting changing `S::CONST` to `<S as A>::CONST`, rather than suggesting a random type parameter `T`.